core/vm/contracts.go

```diff
@@ -474,7 +474,11 @@ func modexpIterationCount(expLen uint64, expHead uint256.Int, multiplier uint64)
 
 	// Add the MSB position - 1 if expHead is non-zero
 	if bitLen := expHead.BitLen(); bitLen > 0 {
-		iterationCount += uint64(bitLen - 1)
+		count, carry := bits.Add64(iterationCount, uint64(bitLen-1), 0)
```

Comment:
`iterationCount = (expLen - 32) * multiplier` also possible to overflow?

expLen can up to maxUint64?

Reply:
Fixed PTAL